### PR TITLE
fix: number of sprints bug

### LINKED ISSUE #53: numberOfSprints in sprint creation isn't actually number of sprints
# Bug Report

## What went wrong?

Tried to create 10 sprints starting at index 7, and instead it just created 7, 8, 9, and 10

## What did you expect should have happened?

Created 7-17 (or at least 16)

## What stacktrace or error message did you see?

none

### PR DESCRIPTION
**Pull request by alancast**
## What did you implement:

Closes #53 

## How did you implement it:

You'll see, it was a simple one

## Todos:

_**Note: Run `npm run check` to run all validation checks on proposed changes**_

- [x] Ensure there are no lint errors
       **Validate via `npm run lint`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
- [x] Write tests and confirm existing functionality is not broken
       **Validate via `npm test`**
- [x] Write documentation
- [x] Pull request title and commit message follows [the conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary)
- [x] Provide verification config / commands / resources
- [x] Requested reviews from at least 2 individuals
- [x] Assigned PR to myself
- [x] Added PR to correct project
- [x] This is ready for review

**_Is it a breaking change?:_** NO

### CODE REVIEW

**Review by elizabethhalper — APPROVED** [review]

**Review by tbarlow12 — APPROVED** [review]
Can we get a unit test to validate this change? Would be good to ensure this doesn't pop up again somehow. Still approved :)

File: src/services/agile/baseAgileService.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export abstract class BaseAgileService implements AgileService {
 
     let currentStartDate = new Date(Date.parse(startDate) + timezoneOffset);
 
-    for (let i = indexStart; i <= numberOfSprints; i++) {
+    for (let i = indexStart; i <= numberOfSprints + indexStart; i++) {
```
Comment (alancast) [inline src/services/agile/baseAgileService.ts:47]: This actually needs to be strictly < because it's starting at index start now

Comment (alancast) [reply src/services/agile/baseAgileService.ts]: fixed

**Comment by alancast** [comment]
@tbarlow12 added unit tests and actually found a bug with the minimal code change lol

**Review by zmmille2 — APPROVED** [review]

**Pull request merged**